### PR TITLE
Add v0.20 release schemas

### PR DESCRIPTION
**Pull request by CyberHoward**
see title

### CODE REVIEW

**Review by h0tw4t3r — CHANGES_REQUESTED** [review]
Can we have a common format of how the contract schema file is named?

Here it's either `module-schema.json` or `${namespace}-${module}.json`.

Is there any reason for them to be different?

File: abstract/abstract-adapter/0.20.0/adapter-schema.json
Comment (dalechyn) [inline abstract/abstract-adapter/0.20.0/adapter-schema.json:1]: the file should be named `abstract-adapter.json`.

File: abstract/abstract-app/0.20.0/app-schema.json
Comment (dalechyn) [inline abstract/abstract-app/0.20.0/app-schema.json:1]: the file should be named `abstract-app.json`.

**Review by dalechyn — CHANGES_REQUESTED** [review]

File: abstract/abstract-adapter/0.20.0/adapter-schema.json
Comment (dalechyn) [reply abstract/abstract-adapter/0.20.0/adapter-schema.json:1]: actually this scheme doesn't work with ts-codegen.

File: abstract/abstract-app/0.20.0/app-schema.json
Comment (dalechyn) [reply abstract/abstract-app/0.20.0/app-schema.json:1]: actually this scheme doesn't work with ts-codegen.

**Pull request merged**